endtoend/profile.test.js

```diff
@@ -0,0 +1,28 @@
+import'babel-polyfill';
+
+const puppeteer = require('puppeteer');
+
+const timeout = 1000;
+
+describe('Profile', () => {
+	let page;
+	let browser
+	beforeAll(async () => {
+		browser = await puppeteer.launch({headless: false});
```

Comment:
Предлагаю запускать тесты в headless режиме :)

```suggestion
		browser = await puppeteer.launch();
```